lib/host.py

```diff
@@ -389,3 +392,13 @@ def join_pool(self, pool):
             lambda: master.xe('host-param-get', {'uuid': self.uuid, 'param-name': 'enabled'}),
             f"Wait for pool {master} to see joined host {self} as enabled."
         )
+
+    def activate_smapi_driver(self, driver):
+        sm_plugins = self.ssh(['grep', '[[:space:]]*sm-plugins[[:space:]]*=[[:space:]]*', XAPI_CONF_FILE]).splitlines()
+        sm_plugins = sm_plugins[-1] + ' ' + driver
+        self.ssh([f'echo "{sm_plugins}" > {XAPI_CONF_DIR}/00-XCP-ng-tests-sm-driver-{driver}.conf'])
```

Comment:
One may argue that the file name template should be a constant, but I'll approve nevertheless.